CMakeLists.txt

```diff
@@ -68,9 +70,9 @@ endif()
 # Install some files to ease package creation
 if(PACKAGE_MODE)
     if(NOT DISABLE_CHEATS_ZIP)
-        INSTALL(FILES     "${CMAKE_SOURCE_DIR}/bin/cheats_ws.zip" DESTINATION "${GAMEINDEX_DIR}")
+        INSTALL(FILES     "${CMAKE_SOURCE_DIR}/bin/cheats_ws.zip" DESTINATION "${CMAKE_INSTALL_DATADIR}/PCSX2")
     endif()
-    INSTALL(FILES     "${CMAKE_SOURCE_DIR}/bin/GameIndex.dbf" DESTINATION "${GAMEINDEX_DIR}")
+    INSTALL(FILES     "${CMAKE_SOURCE_DIR}/bin/GameIndex.dbf" DESTINATION "${CMAKE_INSTALL_DATADIR}/PCSX2")
```

Comment:
_almost certain_ none of those assets would get parsed by PCSX2

Reply:
See the changed lines in ` cmake/BuildParameters.cmake`, they are passed as defines using `CMAKE_INSTALL_FULL_<dir>`.

Reply:
Ah right, makes sense!